.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,233 @@
+name: benchmarks
```

Comment:
That's how mergify works when there is an automated backport, and includes the `conflict` label:
- https://github.com/elastic/apm-server/pull/13344#event-13057794657

In this particular case, `.github/workflows/benchmarks.yml` was not backported originally in the past, but since it was changes among some other things in https://github.com/elastic/apm-server/issues/13307, when backporting it does create the file from scratch with the `conflict` label.

We can delete it or keep it - it won't run for `7.17` at all, `schedule` only works for the default branch, in this particular case `main`.